### ISSUE TITLE
Accessibility: Provide better user interface for the settings screen

### ISSUE THREAD

**Issue by ajborka** [microsoft/vscode#94525]
@isidorn , @roblourens Screen reader users find it difficult to understand how the settings TOC and the settings tree relate to each other. In a typical application that has settings, each settings TOC entry has its own page of settings. Once those settings are visible on the screen, the user must change to a different category in the settings TOC to view another group of settings. Visual Studio 2019 is a prime example of how settings with a treeview of categories should work. For a web-based UI for settings, take a look at Firefox and Chrome. Chrome has the better methodology because it consistantly keeps each group of settings a concern of its own. In other words, vscode needs to ensure users can understand where they are positioned in the settings hierarchy. Unfortunately, vscode does not. To reproduce, do the following:

* Start Orca, JAWS, VoiceOver, or NVDA.
* Start vscode.
* open settings.
* Put focus on the settings TOC.
* Navigate to a settings group such as commonly used or editor>cursor.
* With a keyboard, navigate to the list of settings on the right side of the screen.
* Navigate the settings tree with the TAB key.
* Try putting focus on the settings TOC and settings tree throughout your experiment.

Actual results: A screen reader can access the settings, but the user quickly loses the ability to understand where they are positioned in the settings hierarchy. It is impossible to find out what settings TOC entry I am navigating settings for, and it is impossible to tell when I arrived at the end of the settings for a TOC group.
Expected results: It should be easy to understand what group of settings I am working with at the time.

**Comment by isidorn** [comment]
@ajborka thanks for providing feedback.
This is somehwat aligned with the other feedback we already got for the Settings UX. We should definetly try to improve this. Here are the other issues
https://github.com/microsoft/vscode/issues?q=is%3Aopen+label%3Aaccessibility+label%3Asettings-editor

However I am not sure if this issue brings something new to the table or should we close it as a duplicate. Leaving this to @roblourens

**Comment by roblourens** [comment]
I think this could be considered a dupe of https://github.com/microsoft/vscode/issues/92485 which I remember now but had forgotten we already had a similar discussion.
